builder/vsphere/driver/vm.go

```diff
@@ -315,6 +317,12 @@ func (vm *VirtualMachine) Clone(ctx context.Context, config *CloneConfig) (*Virt
 		configSpec.DeviceChange = append(configSpec.DeviceChange, config)
 	}
 
+	vAppConfig, err := vm.updateVAppConfig(config.VAppProperties)
```

Comment:
Nit: Do you think it would be a good idea to pass `ctx` down here ? I see a few `Properties` call that could take it, and therefore be cancellable by user 🤔  ( if it sounds like premature optimisation please ignore me ! 👍  )

Reply:
That's a good idea!